src/components/SpruceForm/LeafyGreenWidgets.tsx

```diff
@@ -38,7 +43,9 @@ export const LeafyGreenTextInput: React.FC<WidgetProps> = ({
         <TextInput
           data-cy={dataCy}
           value={value === null || value === undefined ? null : `${value}`}
-          label={label}
+          // @ts-ignore
```

Comment:
nit. Can we use `@ts-expect-error` instead. If leafygreen every fixes this we will get an alert and we can remove the  statement.